xclim/indices/_multivariate.py

```diff
@@ -1804,7 +1812,8 @@ def winter_rain_ratio(
     """
     ratio = liquid_precip_ratio(pr, prsn, tas, freq=freq)
     winter = ratio.indexes["time"].month == 12
-    return ratio.sel(time=winter)
+    out: xarray.DataArray = ratio.sel(time=winter)
```

Comment:
Doesn't xarray already provide complete typing ? Isn't the output of `DataArray.sel` already annotated `DataArray` ?